index.js

```diff
@@ -83,38 +93,38 @@ function attachmentGif(w) {
 }
 
 function loadWords() {
-    persistence.load(function(remainingWords, bingoedWords, err){
+    persistence.load(function (remainingWords, bingoedWords, err) {
         if (err == undefined) {
             console.log('Loaded saved words successfully');
             foundWords = bingoedWords.filter(w => {
-                w.regExp= makeRegex(w.word);
+                w.regExp = makeRegex(w.word);
                 return true;
             });
             words = remainingWords.filter(w => {
-                w.regExp= makeRegex(w.word);
+                w.regExp = makeRegex(w.word);
                 return true;
             });
         } else {
             console.log("Load words from initial.txt");
             readFile('initial.txt', 'utf8').then(txt => {
-                Promise.map(txt.split(/[\n\r]+/), w => addWord(w), {concurrency: 5})
+                Promise.map(txt.split(/[\n\r]+/), w => addWord(w), {
+                    concurrency: 5
+                })
             })
         }
     });
 }
 
 function exitHandler() {
-    persistence.save(words, foundWords, function(err) {
+    bot.destroy(bot);
+    persistence.save(words, foundWords, function (err) {
         if (err) {
             throw err;
         } else {
             console.log('Wordlist and found words saved successfully.');
             process.exit(0);
         }
     });
-    if (typeof  bot.destroy === 'function') {
```

Comment:
This has to stay in this position here. When `bot.destroy()` is not a method the saving of the words will also fail. And that's not that what we want.